### PR TITLE
Minor edits, `Knowledgebase` -> `Docs` throughout

### LINKED ISSUE #89: Check URLs work
Check polkadot wiki links - for example: https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/1084/files.

Verify all other links.

### PR DESCRIPTION
**Pull request by sacha-l**
Closes #89, #139 and a couple items for #158.

Seems like a lot of #158 will mostly be resolved by fixing the Parachain how-to guides ( [#179](https://github.com/substrate-developer-hub/substrate-docs/issues/179) ).

### CODE REVIEW

**Comment by lsgunnlsgunn** [comment]
Docs is fine for now. I prefer it over Knowledge Base.

**Review by NukeManDan — APPROVED** [review]

File: src/components/layout/homepage/CommunityCard.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default function CommunityCard() {
     {
       title: 'Hackathons & Grants',
       description:
-        'Jump-start your project or your career in the blockchain space',
+        'Find out how you can jump-start your project or your career in the blockchain space.',
```
Comment (jimmychu0807) [inline src/components/layout/homepage/CommunityCard.tsx:37]: I actually prefer the previous, more concise version. It has the same meaning but a stronger voice by starting with a verb.

File: v3/docs/01-getting-started/e-glossary/index.mdx [hunk 2]
```diff
@@ -564,7 +564,7 @@ The logic of a [blockchain](#blockchain) that determines how the state changes w
 ## Storage Items
 
 [FRAME](#frame) primitives that provide type-safe data persistence capabilities to the
-[runtime](#runtime). Learn more about storage items in the Knowledge Base article for
+[runtime](#runtime). Learn more about storage items in the article about
```
Comment (jimmychu0807) [inline v3/docs/01-getting-started/e-glossary/index.mdx:567]:
```suggestion
[runtime](#runtime). Learn more about storage items in this article about
```

**Review by jimmychu0807 — APPROVED** [review]

File: src/components/layout/homepage/CommunityCard.tsx (hunk 1 above)
Comment (sacha-l) [inline src/components/layout/homepage/CommunityCard.tsx:37]:
```suggestion
        'Jump-start your project or your career in the blockchain space.',
```

**Comment by sacha-l** [comment]
The 2 broken links are hackMD URLs that we'll fix in later updates =)

**Pull request merged**